src/main/scala/com/timeout/KinesisGraphStage.scala

```diff
@@ -70,6 +74,26 @@ class KinesisGraphStage[A : ToPutRecordsRequest](putRecords: PutRecords, streamN
       }
     }
 
+    /**
+      * Split
+      */
+    private def splitBySize(entries: List[PutRecordsRequestEntry]): List[List[PutRecordsRequestEntry]] = {
```

Comment:
That will work if and only if each record size < `maxRequestSize`. If not, then our request will be rejected by aws anyway.

Reply:
Were you meant to say the request size < maxRequestSize